src/module_2/EDA_1.md

```diff
@@ -0,0 +1,3314 @@
+    1958           NaN             NaN           NaN         NaN  
+    675            NaN             NaN           NaN         NaN  
+    4694           NaN             NaN           NaN         NaN  
+
+
+
+```python
+# Verificar valores faltantes
+print("\nValores faltantes en Orders:")
+print(orders.isnull().sum())
+print("\nValores faltantes en Regulars:")
+print(regulars.isnull().sum())
+print("\nValores faltantes en Abandoned Cart:")
+print(abandoned_cart.isnull().sum())
+print("\nValores faltantes en Inventory:")
+print(inventory.isnull().sum())
+print("\nValores faltantes en Users:")
+print(users.isnull().sum())
+
+```
+
+    
+    Valores faltantes en Orders:
+    id                0
+    user_id           0
+    created_at        0
+    order_date        0
+    user_order_seq    0
+    ordered_items     0
+    dtype: int64
+    
+    Valores faltantes en Regulars:
+    user_id       0
+    variant_id    0
+    created_at    0
+    dtype: int64
+    
+    Valores faltantes en Abandoned Cart:
+    id            0
+    user_id       0
+    created_at    0
+    variant_id    0
+    dtype: int64
+    
+    Valores faltantes en Inventory:
+    variant_id          0
+    price               0
+    compare_at_price    0
+    vendor              0
+    product_type        0
+    tags                0
+    dtype: int64
+    
+    Valores faltantes en Users:
+    user_id                     0
+    user_segment                0
+    user_nuts1                 51
+    first_ordered_at            0
+    customer_cohort_month       0
+    count_people             4658
+    count_adults             4658
+    count_children           4658
+    count_babies             4658
+    count_pets               4658
+    dtype: int64
+
+
+
+```python
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+# Define la ruta donde están los archivos Parquet
+data_path = '/home/eduardo_1999/projects/zrive-ds/projects/zrive-ds/sampled-datasets/'
+
+# Cargar los archivos Parquet
+orders = pd.read_parquet(data_path + 'orders.parquet')
+regulars = pd.read_parquet(data_path + 'regulars.parquet')
+abandoned_cart = pd.read_parquet(data_path + 'abandoned_carts.parquet')  
+inventory = pd.read_parquet(data_path + 'inventory.parquet')
+users = pd.read_parquet(data_path + 'users.parquet')
+
+# Sanity Checks
+print("Shapes of datasets:")
+print("Orders:", orders.shape)
+print("Regulars:", regulars.shape)
+print("Abandoned Cart:", abandoned_cart.shape)
+print("Inventory:", inventory.shape)
+print("Users:", users.shape)
+
+# Revisar tipos de datos y valores nulos
+print("\nData Types and Missing Values:")
+for df in [orders, regulars, abandoned_cart, inventory, users]:
+    print(df.info())
+
+# Análisis de la tasa de abandono del carrito
+abandoned_cart[''] = abandoned_cart['user_id'].apply(lambda x: len(x))  # Contar artículos abandonados
+sns.histplot(abandoned_cart['item_ids'], bins=20)
```

Comment:
I'd recomment to create an axes first `fig, ax = plt.subplots()` and then pass it to seaborn. It gives you more control on the matplotlib operations